### PR TITLE
[DDW-737] Unify CPU info in diagnostics dialog

### PR DESCRIPTION
**Pull request by szymonmaslowski**
This PR unifies CPU info in diagnostics dialog.

⚠️ Apart from the diagnostics dialog, same unification logic is applied to the State Snapshot Logging mechanism in the [ProfileStore](https://github.com/input-output-hk/daedalus/pull/2818/files#diff-24a9886e3521b1466b353e75c22a198ec77669bc439e12024122ddb5e8540bda).

Unfortunately it is not possible to obtain the info about number of cores. The data we deal with is in form of an array where each item represents a single processor's thread (https://nodejs.org/api/os.html#oscpus). However there is no information about how many threads per core that particular processor have so we cannot calculate it.

The target format is `<processor model name> @ <speed>GHz`. However if somehow there won't be enough data to produce that format result could contain just the processor model name or an empty string if even the model couldn't be obtained.

## Screenshots

image
image

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1642416681043800)
- [ ] Test

---

### Basics
- [x] PR assigned to the PR author(s) 
- [ ] `input-output-hk/daedalus-dev` and `input-output-hk/daedalus-qa` assigned as PR reviewers
- [ ] If there are UI changes, Alexander Rukin assigned as an additional reviewer
- [x] PR has appropriate labels (`release-vNext`, `feature`/`bug`/`chore`, `WIP`)
- [x] PR link is added to a Jira ticket, ticket moved to In Review
- [x] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [x] PR has a good description that summarizes all changes
- [x] PR contains screenshots (in case of UI changes)
- [x] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [x] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [x] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [x] Important parts of the code are properly commented and documented
- [x] Code is properly typed with flow
- [x] React components are split-up enough to avoid unnecessary re-renderings
- [x] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

File: source/renderer/app/components/status/formatCpuInfo.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// @flow
+import type { Cpu } from '../../types/systemInfoTypes';
+import { formattedNumber } from '../../utils/formatters';
+
+const formatCpuInfo = ({ model, speed }: Cpu): string => {
+  const modelName = model.split('@')[0].trim().replace(/\s+/g, ' ');
+  const speedValue = formattedNumber(speed / 1000, 2);
+
+  return `${modelName} @ ${speedValue}GHz`;
```
Comment (marcin-mazurek) [inline source/renderer/app/components/status/formatCpuInfo.js:9]: Do we need to take into account cases where either model name or speed is not going to be available? To avoid something like: `MyUnknownProcessor @ 0GHz`?

File: source/renderer/app/components/status/formatCpuInfo.spec.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import formatCpuInfo from './formatCpuInfo';
+
+describe('Formatters/formatCpuInfo', () => {
+  it('gives correct model name and clock speed for Intel format', () => {
+    expect(
```
Comment (marcin-mazurek) [inline source/renderer/app/components/status/formatCpuInfo.spec.js:5]: Do you think it would be beneficial to add a test case for edge cases eg. when one of the params is missing?

File: source/renderer/app/components/status/formatCpuInfo.js (hunk 1 above)
Comment (szymonmaslowski) [reply source/renderer/app/components/status/formatCpuInfo.js:9]: Probably such case would be impossible, however I agree. Let's be more defensive. I will consider three cases:
1. No model property - return empty string
2. No speed - return only formatted model name.
3. return full information

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,10 @@
 
 ## vNext
 
+### Features
+
+- Unify CPU info in diagnostics dialog ([PR 2818](https://github.com/input-output-hk/daedalus/pull/2818))
```
Comment (lucas-barros) [inline CHANGELOG.md:7]: We should use past tense here "Unified"

Comment (szymonmaslowski) [reply CHANGELOG.md]: Fixed

**Review by lucas-barros — APPROVED** [review]

**Review by dmitrii-gaico — APPROVED** [review]
Looks great ! Validated at Win / Lin / Mac. Thanks!

File: source/renderer/app/utils/formatters.js [hunk 4]
```diff
@@ -260,24 +260,6 @@ export const toFixedWithoutRounding = (
 export const formattedNumber = (value: number | string, dp?: number): string =>
   new BigNumber(value).toFormat(dp);
 
-export const formattedCpuModel = (model: string): string => {
```
Comment (renanvalentin) [inline source/renderer/app/utils/formatters.js:263]: [optional] Why moving `formattedCpuModel` out of `formatters.js`? 

I notice there's a  `formattedBytesToSize` that's related to profiling as well. So i wonder if `formatCpuInfo` should be moved back to formatters.js to keep similar functions grouped.

**Review by renanvalentin — APPROVED** [review]
